producers-commissioning/src/unit-test/java/uk/gov/moj/sdt/producers/comx/config/submitquery/SubmitQueryConfigMcolDefenceFourTest.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.moj.sdt.producers.comx.config.submitquery;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.moj.sdt.domain.ErrorLog;
+import uk.gov.moj.sdt.domain.SubmitQueryRequest;
+import uk.gov.moj.sdt.domain.api.IErrorLog;
+
+class SubmitQueryConfigMcolDefenceFourTest extends SubmitQueryConfigTestBase {
+
+    private SubmitQueryConfigMcolDefenceFour submitQueryConfigMcolDefenceFour;
+
+    @Override
+    protected void setUpLocalTests() {
+        submitQueryConfigMcolDefenceFour = new SubmitQueryConfigMcolDefenceFour();
+    }
+
+    @Test
+    void testSubmitQueryRequest() {
```

Comment:
this test seems to be repeated, can we remove duplicates

Reply:
The SubmitQueryConfig classes don't have a common relation between them (either implementing a common interface or sharing a superclass), so unfortunately we can't write a single set of common tests and pass them in to it as parameters.